axon-server-connector/src/test/java/org/axonframework/axonserver/connector/command/GrpcBackedCommandMessageTest.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2010-2019. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.axonserver.connector.command;
+
+import io.axoniq.axonserver.grpc.command.Command;
+import org.axonframework.axonserver.connector.AxonServerConfiguration;
+import org.axonframework.commandhandling.CommandMessage;
+import org.axonframework.commandhandling.GenericCommandMessage;
+import org.axonframework.messaging.MetaData;
+import org.axonframework.serialization.Serializer;
+import org.axonframework.serialization.xml.XStreamSerializer;
+import org.junit.*;
+
+import java.util.Objects;
+
+import static org.junit.Assert.*;
+
+public class GrpcBackedCommandMessageTest {
```

Comment:
Short JavaDoc explaining what is being tested?

Reply:
Sure thing.